release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -0,0 +1,896 @@
+
+In the table, features are marked with the following statuses:
+
+* *GA*: _General Availability_
+* *DEP*: _Deprecated_
+* *REM*: _Removed_
+
+//TODO: remove anything that has been REM since 4.7?
+
+.Deprecated and removed features tracker
+[cols="3,1,1,1",options="header"]
+|====
+|Feature |OCP 4.9 |OCP 4.10 | OCP 4.11
+
+|SQLite database format for Operator catalogs
+|DEP
+|DEP
+|DEP
+
+|`ImageChangesInProgress` condition for Cluster Samples Operator
+|DEP
+|DEP
+|DEP
+
+|`MigrationInProgress` condition for Cluster Samples Operator
+|DEP
+|DEP
+|DEP
+
+|Cluster Loader
+|DEP
+|REM
+|REM
+
+|Bring your own {op-system-base} 7 compute machines
+|DEP
+|REM
+|REM
+
+|Jenkins Operator
+|DEP
+|REM
+|REM
+
+|Grafana component in monitoring stack
+|-
+|DEP
+|REM
+
+|Access to Prometheus and Grafana UIs in monitoring stack
+|
+|DEP
+|REM
+
+|vSphere 6.7 Update 2 or earlier
+|DEP
+|DEP
+|REM
+
+|vSphere 7.0 Update 1 or earlier
+|-
+|-
+|DEP
+
+|Virtual hardware version 13
+|DEP
+|DEP
+|REM
+
+|VMware ESXi 6.7 Update 2 or earlier
+|DEP
+|DEP
+|REM
+
+|VMware ESXi 7.0 Update 1 or earlier
+|-
+|-
+|DEP
+
+|Snapshot.storage.k8s.io/v1beta1 API endpoint
+|DEP
+|DEP
+|REM
+
+|Minting credentials for Microsoft Azure clusters
+|GA
+|REM
+|REM
+
+|Persistent storage using FlexVolume
+|-
+|DEP
+|DEP
+
+|Automatic generation of service account token secrets
+|GA
+|GA
+|REM
+
+|Removal of Jenkins images from install payload
+|GA
+|GA
+|REM
+
+|====
+
+[id="ocp-4-12-deprecated-features"]
+=== Deprecated features
+
+[id="ocp-4-12-oc-commands-flags-tokens-deprecated"]
+==== OpenShift CLI (oc) commands and flags for requesting tokens are deprecated
+
+The following `oc` commands and flags for requesting tokens are now deprecated:
+
+* The `oc serviceaccounts create-kubeconfig` command
+* The `oc serviceaccounts get-token` command
+* The `oc serviceaccounts new-token` command
+* The `--service-account/-z` flag for the `oc registry login` command
+
+Use the `oc create token` command instead to request tokens.
+
+[id="ocp-4-12-rhv-deprecations"]
+==== Red Hat Virtualization (RHV) as a host platform for {product-title} will be deprecated
+
+Red Hat Virtualization (RHV) will be deprecated in an upcoming release of {product-title}. Support for {product-title} on RHV will be removed from a future {product-title} release, currently planned as {product-title} 4.14.
+
+[id="ocp-4-12-deprecated-features-vsphere-7.0update1"]
+==== Support for vSphere 7.0 Update 1 or earlier is deprecated
+In {product-title} 4.11, support for VMware vSphere 7.0 Update 1 or earlier is deprecated. While vSphere 7.0 Update 1 or earlier remains fully supported, Red Hat recommends that you use vSphere 7.0 Update 2 or later.
```

Comment:
There are a lot of remaining 4.11 references in the next ~75 lines. Did you want to retain these?

Reply:
See above for deciding to update later.